trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/services/DataProviderService.java

```diff
@@ -1101,6 +1105,70 @@ private Response getTree(UUID expUUID, String outputId, QueryParameters queryPar
         }
     }
 
+    /**
+     * Query the provider for an image-based data provider
+     *
+     * @param expUUID
+     *            desired experiment UUID
+     * @param outputId
+     *            Output ID for the data provider to query
+     * @return {@link Response} with the corresponding image
+     */
+    @GET
+    @Path("/image/{outputId}")
```

Comment:
I'm debating with myself it this is the right endpoint. `/image/` for images per experiment. We would we have then other endpoints for other types, e.g. `/pdf/` or `/html` (multipart) or others.  See also comment on 
https://github.com/eclipse-tracecompass/org.eclipse.tracecompass/pull/242#issuecomment-2783260628